Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -49,11 +50,10 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.goobi.production.cli.helper.WikiFieldHelper;
-import org.goobi.production.constants.Parameters;
+import org.goobi.production.plugin.PluginLoader;
```

Comment:
Wrong imports order.

Reply:
This is just the code formatter reformatting the comment, I didn't change it.